zrml/rikiddo/src/types/sigmoid_fee.rs

```diff
@@ -0,0 +1,72 @@
+use crate::{
```

Comment:
Can you write a quick root doc (`//*`) about the sigmoid fee and its responsibility on Rikiddo?
I known about the white-paper and other documentations but something small with one or two lines would benefit other developers that are seeing this code for the first time

Reply:
A completed documentation will be added at #217 , is it okay from your perspective to leave it like this until then?

Reply:
OK. Didn't see the issue.